srcpkgs/Shortwave/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'Shortwave'
+pkgname=Shortwave
+version=2.0.1
+revision=1
+build_style=meson
+build_helper=rust
+hostmakedepends="cargo gettext git glib-devel pkg-config rust sqlite-devel"
+makedepends="gstreamer1-devel gst-plugins-bad1-devel gtk4-devel libadwaita-devel openssl-devel rust-std sqlite-devel"
+depends="gst-plugins-good1"
+checkdepends="desktop-file-utils"
+short_desc="GTK internet radio player, written in Rust"
+maintainer="Philipp David <pd@3b.pm>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/World/Shortwave"
+distfiles="https://gitlab.gnome.org/World/${pkgname}/-/archive/${version}/${pkgname}-${version}.tar.gz"
+checksum=65dd02f7ad0b286613eae5d2f86adf9e8725ddc7885dd8658b2863cf13c6e594
+
+post_patch() {
+	[ -z "$CROSS_BUILD" ] && return 0
+	vsed -i build-aux/cargo.sh \
+		-e 's%"$CARGO_TARGET_DIR"/%&${RUST_TARGET}/%' \
+		-e '/CARGO_HOME/d'
```

Comment:
Style nit, I find `if [ "$CROSS_BUILD" ]; then [vsed command] fi` cleaner to understand.

Reply:
I took this snippet straight from the gnome-podcasts template, but that really is cleaner.